Python-Input.md

```diff
@@ -0,0 +1,33 @@
+# Python Input() Function
+
+Many a times, in a program we need some input from the user. Taking inputs from the user makes the program feel a bit interactive. In python, to take input from the user we have a function `input()`. Let's see some examples:
+
+1. When we just want to take the input:
+```python
+# This will just give a prompt without any message
+inp = input()
+```
+[Run Code](https://repl.it/CUqX/0)
```

Comment:
@alayek keep it here, users can test the working of `input`. however @meetmangukiya , edit the snippet to have a `print()` statement to display what the user had given. And no, you dont need to squash the commits, we can do it for you!